### DISCUSSION TITLE
Send SMS - sca field, what is it for?

### DISCUSSION THREAD

**Discussion by lukeatkins** [Salamek/huawei-lte-api#160] [Q&A] [answered]
Hello,
Does anyone know what the sca field is for when sending an SMS?
I thought maybe it was the caller Id, but I can't seem to make it work like that.

Is there any way to specify an different caller Id so that replies are disabled?

**Comment by Salamek** [comment, marked as answer]
Hi SCA field is a phone number of your operator message center number in INTL format e.g. +420603052000
I have added comments in Sms class to clear this out

Also i don't  know any way to specify a caller ID
